sleap_nn/inference/topdown.py

```diff
@@ -469,38 +515,20 @@ def forward(self, batch: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor]:
             `"pred_peak_vals": (batch_size, n_nodes)`: Confidence
                 values for the instance skeleton points.
         """
-        batch_size = batch["video_idx"].shape[0]
+        if isinstance(self.instance_peaks, FindInstancePeaksGroundTruth):
+            if "instances" not in batch:
+                raise ValueError(
+                    "Ground truth data was not detected... "
+                    "Please load both models when predicting on non-ground-truth data."
+                )
```

Comment:
Let's have an issue that describes the case where we have a single-node skeleton and we're doing centroid-only inference?